scripts/run-tests.ts

```diff
@@ -2,21 +2,25 @@ import * as path from 'path';
 import { globP, forkP } from './utils';
 
 export interface Options {
-  globs: string[],
+  includeGlobs: string[],
+  excludeGlobs?: string[],
   rootDir: string,
   coverage?: boolean,
   timeout?: number
 }
 
 export async function runTests(options: Options) {
-  const { globs, rootDir, coverage, timeout } = options;
+  const { includeGlobs, excludeGlobs, rootDir, coverage, timeout } = options;
   function nodule(nodule: string) {
     return path.relative(rootDir, `node_modules/${nodule}`);
   }
   let testNum = 0;
-  const files = ([] as string[])
-    .concat(...await Promise.all(globs.map(glob => globP(glob))))
-    .filter(_ => true);
+  const excludedFiles = ([] as string[])
```

Comment:
I think this is better than type-casting:
```ts
const excludedFiles: string[] = [].concat(...);
```

Reply:
That syntax doesn't work for TS. `[]` must be known to be a `string[]` first.

An alternative is

```ts
const excludedFiles: string[] = [];
excludedFiles.concat(...);
```

But I think that's a little more cumbersome.